iocage/Release.py

```diff
@@ -215,6 +215,11 @@ class ReleaseGenerator(ReleaseResource):
         "sendmail_submit_enable": False,
         "sendmail_msp_queue_enable": False,
         "sendmail_outbound_enable": False,
+        "daily_clean_hoststat_enable": False,
+        "daily_status_mail_rejects_enable": False,
+        "daily_status_include_submit_mailq": False,
+        "daily_submit_queuerun": False,
```

Comment:
i don't know how i got the idea to put these into `rc.conf`, but they belong into [`periodic.conf`](https://www.freebsd.org/cgi/man.cgi?periodic.conf)